build/premake/premake.fixes.lua

```diff
@@ -0,0 +1,20 @@
+-- https://github.com/premake/premake-core/issues/1559
```

Comment:
Please add _HACK_ here.

Reply:
Do you want me to rename the file to premake.hacks instead?